specifications/transactions/ping-command/README.md

```diff
@@ -0,0 +1,18 @@
+# PingCommand
+Th `PingCommand` primarly provides a robust method for healthcheck for an off-chain service. In addition, its simplicity makes it an ideal candidate as the first milestone for validating a correctly implemented off-chain service. Upon receipt of a `PingCommand` the recipient responsds with an empty `CommandResponseObject` completing the exchange. Hence this protocol has no requirements for persistency, so long as the remote service is available and online, it is expected to respond.
+
+`PingCommand` Object:
+```
+{
+    "_ObjectType": "CommandRequestObject",
+    "command_type": "PingCommand",
+    "command": {
+	    "_ObjectType": "PingCommand",
+    },
```

Comment:
I mean technically this could be empty, but I don't have a strong opinion, especially if we do want to add a payload later.